Formula/n/netdata.rb

```diff
@@ -21,84 +20,74 @@ class Netdata < Formula
     sha256 x86_64_linux:   "eaf7d9f20de88f1e860c8f3709cb98d9d0cc37dc45f200d2d5e1df2fb6bdaf48"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "m4" => :build
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
+  depends_on "brotli"
+  depends_on "freeipmi"
   depends_on "json-c"
   depends_on "libuv"
   depends_on "libyaml"
   depends_on "lz4"
   depends_on "openssl@3"
   depends_on "pcre2"
-  depends_on "protobuf"
-  depends_on "protobuf-c"
+  depends_on "zstd"
 
   uses_from_macos "zlib"
 
   on_linux do
     depends_on "util-linux"
   end
 
-  resource "judy" do
-    url "https://downloads.sourceforge.net/project/judy/judy/Judy-1.0.5/Judy-1.0.5.tar.gz"
-    sha256 "d2704089f85fdb6f2cd7e77be21170ced4b4375c03ef1ad4cf1075bd414a63eb"
-  end
+  patch :DATA
```

Comment:
Is this submitted upstream?

Reply:
Not yet, but planning to! I should probably keep this PR in Draft for now, it's not really review-ready.

Reply:
https://github.com/netdata/netdata/issues/17242